src/ui/utils.js

```diff
@@ -478,3 +493,31 @@ function getTableCellAtPosition( position ) {
 
 	return isTableCellSelected ? position.nodeAfter : findAncestor( 'tableCell', position );
 }
+
+// Returns bounding rect for list of rects.
+//
+// @param {Array.<module:utils/dom/rect~Rect>|Array.<*>} list List of `Rect`s or any list to map by `mapFn`.
+// @param {Function} [mapFn] Mapping function for list elements.
+// @returns {module:utils/dom/rect~Rect}
+function createBoundingRect( list, mapFn = null ) {
+	const rectData = {
+		left: Number.POSITIVE_INFINITY,
+		top: Number.POSITIVE_INFINITY,
+		right: Number.NEGATIVE_INFINITY,
+		bottom: Number.NEGATIVE_INFINITY
+	};
+
+	for ( const item of list ) {
+		const rect = mapFn ? mapFn( item ) : item;
```

Comment:
Coverage dropped and this else branch isn't covered.

![](https://user-images.githubusercontent.com/247363/77895070-84ec2f80-7276-11ea-8c36-ddd664d86aa3.png)

Reply:
Please check if tests needed update or if the branches are really needed.